src/status_im/contexts/wallet/signals.cljs

```diff
@@ -37,9 +37,10 @@
        "wallet-blockchain-status-changed"         {:fx [[:dispatch
                                                          [:wallet/blockchain-status-changed
                                                           (transforms/js->clj event-js)]]]}
-       "wallet-activity-filtering-done"           {:fx [[:dispatch
-                                                         [:wallet/activity-filtering-done
-                                                          (transforms/js->clj event-js)]]]}
+       "wallet-activity-filtering-done"           {:fx
+                                                   [[:dispatch
+                                                     [:wallet/activity-filtering-for-current-account-done
+                                                      (transforms/js->clj event-js)]]]}
```

Comment:
Changed because now (temporary) we process activites per account, not for all the accounts